content/posts/2018-06-16-morning-fun.md

```diff
@@ -0,0 +1,184 @@
+sphere are immediately thrown out of the window as _non-essential_ since
+there is no liability for them.
+
+This effect can be related not only to the development-operations cycle
+but to the development-QA one. If the development team does no QA procedures
+and delegates this task to the QA team it will lead to unforeseen consequences.
+The worst case scenario — developers would not test their code at all
+since in their minds it is not their duty. The best case — corner cases
+would not be checked by developers,
+leading to potential under-development of the scope.
+
+This is where we come back to Amazon.
+
+> The traditional model is that you take your software to the wall
+> that separates development and operations, and throw it over and then forget about it.
+> Not at Amazon. _You build it, you run it_. This brings developers into contact
+> with the day-to-day operation of their software.
+>
+> — [_Werner Vogels_](https://queue.acm.org/detail.cfm?id=1142065)
+
+DevOps culture is all about that. Involving developers in the running cycle
+[brings a lot of benefits](https://aws.amazon.com/blogs/enterprise-strategy/enterprise-devops-why-you-should-run-what-you-build/),
+such as systems design focused on production, transparency, efficient automation
+and tighter customer-developer feedback loop.
+
+Not everyone shares this belief though. Especially developers. Of course!
+It requires extra work, precision and effectiveness. But let’s face a fact —
+if there is an anxiety about being on-call and in the running cycle,
+there is definitely something not healthy with the overall process.
+
+> On-call isn’t a silo. It’s in many ways the microcosm of the engineering skills
+> of an organization (resilience of the systems being built as well as
+> the quality of monitoring, alerting and automation) which in turn is
+> a reflection of the quality of management and prioritization (engineering culture).
+>
+> — [_Cindy Sridharan_](https://medium.com/@copyconstruct/on-call-b0bd8c5ea4e0)
+
+# Going Mobile
+
+The thing is, mobile developers already live in the _You Build It — You Run It_ reality.
+Mostly because such teams are usually small or medium-sized.
+There is no infrastructure needed to be run since
+Google and Apple do that themselves. Plus, it is rare to see a dedicated support team
+responding to users. As a result, the development-running loop is pretty tight.
+Developers often oversee the development process itself, deployment and running.
+
+At the same time, [we can do better](https://www.youtube.com/watch?v=SLILjDx0SO0).
+
+# On-call
+
+> :book: Reading material on the topic:
+> [_Testing in Production_](https://medium.com/@copyconstruct/testing-in-production-the-safe-way-18ca102d0ef1) and
+> [_On-call Doesn’t Have to Suck_](https://medium.com/@copyconstruct/on-call-b0bd8c5ea4e0).
+
+Internally we call it _Daily Duty_ because...
+
+* On-call actions are performed in a 24-hour window.
```

Comment:
So, this is where people like me would be heated up :)

You can clarify that in your case it doesn't mean that you need to react at night or on weekends.

Reply:
Mentioned it, thanks!